Frontend_Application/src/routes/login/login.component.jsx

```diff
@@ -6,13 +7,82 @@ import Typography from "@mui/material/Typography";
 import TextField from "@mui/material/TextField";
 import Button from "@mui/material/Button";
 import Link from "@mui/material/Link";
+import Snackbar from "@mui/material/Snackbar";
+import Alert from '@mui/material/Alert';
+import { Link as RouterLink } from 'react-router-dom';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import IconButton from "@mui/material/IconButton";
 
 const Login = () => {
 
+  const [formData, setFormData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const [errors, setErrors] = useState({
+    email: '',
+    password: '',
+  });
+
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState('');
+
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+    // Clear error when input changes
+    setErrors({ ...errors, [e.target.name]: '' }); 
+  };
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+
+    const newErrors = {};
+
+    // Validation
+    if (!formData.email.trim()) {
+      newErrors.email = 'Email is required';
+    } else if (!formData.email.includes('@') || !formData.email.endsWith('.com')) {
```

Comment:
Have we considered emails that don't end with .com?

I have  a .edu email for grad school and my personal email ends in .xyz.

Reply:
Good catch!